### PR TITLE
[MINOR] Add comment for how to test GlutenBroadcastJoinSuite

### PR DESCRIPTION
**Pull request by PHILO-HE**

### CODE REVIEW

**Comment by PHILO-HE** [comment]
Need some refinement to avoid largely extending the time of some CI jobs.

File: .github/workflows/velox_be.yml [hunk 1]
```diff
@@ -180,7 +174,7 @@ jobs:
       - name: Build and Run unit test for Spark 3.3.1(slow tests)
         run: |
           docker exec velox-ubuntu2004-test-spark33-slow-$GITHUB_RUN_ID bash -l -c 'cd /opt/gluten && \
-          mvn clean install -Pspark-3.3 -Pbackends-velox -Prss -Pspark-ut -DargLine="-Dspark.test.home=/var/cache/spark33/spark331" -DtagsToInclude=org.apache.spark.tags.ExtendedSQLTest'
+          mvn clean install -Pspark-3.3 -Pbackends-velox -Prss -Pspark-ut -DargLine="-Dspark.test.home=/opt/spark322" -DtagsToInclude=org.apache.spark.tags.ExtendedSQLTest'
```
Comment (zhouyuan) [inline .github/workflows/velox_be.yml:177]: I'm not sure on this, as there are some difference on spark33 vs spark32 on unit tests

Comment (PHILO-HE) [reply .github/workflows/velox_be.yml:177]: It looks the new test failure comes from the inconsistent spark test home. So we cannot do this.
As mentioned in the meeting, it is better to directly use the pre-built spark kept in docker image, which can not only fix this test issue, but also accelerate the CI test.

**Comment by PHILO-HE** [comment]
For occasional test failure of GlutenBroadcastJoinSuite, the fix will be covered by https://github.com/oap-project/gluten/pull/2869.

**Review by rui-mo — APPROVED** [review]

**Review by rui-mo — APPROVED** [review]

**Pull request merged**